app/gui2/src/stores/graph/index.ts

```diff
@@ -248,10 +248,6 @@ export const useGraphStore = defineStore('graph', () => {
     unconnectedEdge.value = undefined
   }
 
-  const method = computed(() =>
-    syncModule.value ? methodAstInModule(syncModule.value) : undefined,
-  )
-
```

Comment:
I saw exactly the same duplication in my code - but I preferred to have computed here, instead of relying on some update being called (so I made `methodAst` this computed). What do you think about it?

Reply:
The `computed` `method` implementation was wrong--module updates are observed by the Y.JS transaction handler, not reactively (it was actually working because of a missing `markRaw`, but I fixed that just now--that is a performance bug, and nothing else relied on it). The `Ref` `methodAst` is updated in `updateState`, when the graph DB is updated, so we can be sure it is always consistent with the displayed graph.